CMakeLists.txt

```diff
@@ -23,7 +25,7 @@ set(RTRLIB_SRC rtrlib/rtr_mgr.c rtrlib/lib/utils.c rtrlib/lib/ip.c rtrlib/lib/ip
 set(RTRLIB_LINK ${RT_LIB} ${CMAKE_THREAD_LIBS_INIT})
 
 find_package(LibSSH 0.6.0)
-if(LIBSSH_FOUND)
+if(LIBSSH_FOUND AND NOT NOSSH)
```

Comment:
searching for the SSH lib could also be disabled by the parameter in cmake/modules/FindLibSSH.cmake